src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -66,7 +70,8 @@ public partial class Resources : ComponentBase, IAsyncDisposable
 
     private readonly CancellationTokenSource _watchTaskCancellationTokenSource = new();
     private readonly ConcurrentDictionary<string, ResourceViewModel> _resourceByName = new(StringComparers.ResourceName);
-    private readonly HashSet<string> _expandedResourceNames = [];
+    private readonly Dictionary<string, bool> _resourceNamesToExpanded = [];
```

Comment:
Since resources are expanded by default, it seems like it would be simpler to invert the collapse logic and variable names to focus on what is collapsed: `_collapsedResourceNames`

This is what the trace detail page does which shows rows expanded by default

Reply:
Changed to collapsed resources. Thanks, that's better